lighthouse-core/audits/time-to-first-byte.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const URL = require('../lib/url-shim');
+const Util = require('../report/v2/renderer/util');
+
+const TTFB_THRESHOLD = 600;
+
+class TTFBMetric extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'time-to-first-byte',
+      description: 'Keep server response times low (TTFB)',
+      informative: true,
+      helpText: 'Time To First Byte identifies the time at which your server sends a response.' +
+        ' [Learn more](https://developers.google.com/web/tools/chrome-devtools/network-performance/issues).',
+      requiredArtifacts: ['devtoolsLogs', 'URL']
+    };
+  }
+
+  static caclulateTTFB(record) {
+    const timing = record._timing;
+
+    return timing.receiveHeadersEnd - timing.sendEnd;
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    const devtoolsLogs = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+
+    return artifacts.requestNetworkRecords(devtoolsLogs)
+      .then((networkRecords) => {
+        let debugString = '';
+
+        const finalUrl = artifacts.URL.finalUrl;
+        const thresholdDisplay = Util.formatMilliseconds(TTFB_THRESHOLD, 1);
+        const finalUrlRequest = networkRecords.find(record => record._url === finalUrl);
+        const ttfb = TTFBMetric.caclulateTTFB(finalUrlRequest);
+        const passed = ttfb < TTFB_THRESHOLD;
+
+        if (!passed) {
+          debugString = `Root document (${URL.getURLDisplayName(finalUrl)}) went over` +
+            ` the ${thresholdDisplay} threshold`;
```

Comment:
if it's over the threshold then we can be confident it took at least that amount of time, so let's report the actual ttfb here e.g. `Root document request took X ms`

Reply:
something like `Root document took ${Util.formatMilliseconds(ttfb, 1)} ms to get the first byte.` or keep it short as you stated above

Reply:
yours sgtm too 👍